mise.toml

```diff
@@ -0,0 +1,9 @@
+[tools]
+
+# Core dependencies
+jq = "1.7.1"
+just = "1.37.0"
+"ubi:kurtosis-tech/kurtosis-cli-release-artifacts[exe=kurtosis]" = "2.1.0"
```

Comment:
note that we're currently stuck at version 1.4.3 as we need the cli to be compatible with the library we use (which in turn is blocked by https://github.com/kurtosis-tech/kurtosis/pull/2635)

Reply:
Will update